ui/pages/routes/routes.component.js

```diff
@@ -301,6 +306,10 @@ export default class Routes extends Component {
     }
   };
 
+  handleClose = () => {
```

Comment:
I think this function can just be defined within the `NewNetworkInfo` popup itself. Because it is only used in that popup, and the prop it depends (`providerChaindId`) is only used in that popup, then the code will be easiest to read and maintain if this is defined within the popup.

As a general rule, we should only pass props from parent to child components if there is some reason the parent component needs access to the prop, or if doing so makes the implementation of the child component a lot easier, or maybe if the child component is meant to be used in multiple places and have behaviour that is dependent on the parent. Otherwise, we can just select or define a prop within the child component,

Reply:
Fixed in: 11eaf89a7c325b3630324c49fef897cb2c4e6a42